recipes/highs/all/conanfile.py

```diff
@@ -1,14 +1,11 @@
 from conan import ConanFile
 from conan.tools.apple import fix_apple_shared_install_name
 from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
-from conan.errors import ConanInvalidConfiguration
 from conan.tools.files import apply_conandata_patches, collect_libs, copy, export_conandata_patches, get
 from conan.tools.microsoft import is_msvc
 from conan.tools.scm import Version
 from os.path import join
 
-required_conan_version = ">=1.53.0"
```

Comment:
We want to keep this, it provides a nicer error for old versions :)

Reply:
Added again